src/less/less/ds6/variables.less

```diff
@@ -288,6 +288,12 @@
 @ds6-icon-red-laser-height: 18px;
 @ds6-icon-window-width: 28px;
 @ds6-icon-window-height: 25px;
+@ds6-icon-confirmation-foreground-width: 21px;
```

Comment:
Still seeing "foreground" in the variable names. Background and foreground versions should have same dimensions like our other icons.

Reply:
Fixed!